### PR TITLE
Updated Xamarin.Essentials: Contacts

### PR DESCRIPTION
**Pull request by dimonovdd**
Added information on Getting All Contacts

Linked [Issue](https://github.com/xamarin/Essentials/issues/1403) and [Pull Request](https://github.com/xamarin/Essentials/pull/1405)

### CODE REVIEW

**Comment by PRMerger9** [comment]
@dimonovdd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by davidbritch** [comment]
@jamesmontemagno Views?

**Comment by PRMerger12** [comment]
@dimonovdd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger17** [comment]
@dimonovdd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by dimonovdd** [comment]
Main [pull request ](https://github.com/xamarin/Essentials/pull/1405) was changed.

**Comment by jamesmontemagno** [comment]
I think we are still awaiting a merge and then a nuget release.

**Comment by dimonovdd** [comment]
@jamesmontemagno 

@mattleibow  is doing a merge at the moment

File: docs/essentials/contacts.md [hunk 1]
```diff
@@ -70,18 +70,62 @@ try
...
+{
+    // Handle exception here.
+}
+```
+
+## Platform Differences
+
+# [Android](#tab/android)
+
+- The `cancellationToken` parameter in the `GetAllAsync` method is only used on UWP
```
Comment (mattleibow) [inline docs/essentials/contacts.md]: We probably don't have to actually mention this.

File: docs/essentials/contacts.md [hunk 2]
```diff
@@ -70,18 +70,62 @@ try
...
+
+# [iOS](#tab/ios)
+
+- The `cancellationToken` parameter in the `GetAllAsync` method is only used on UWP
+- The ios platform does not have the DiasplayName implementation, so the `Contact.DiasplayName` is always `GivenName + FamilyName`
```
Comment (mattleibow) [inline docs/essentials/contacts.md]:
```suggestion
- The iOS platform does not support the `DisplayName` property natively, thus the `DisplayName` value is constructed as "GivenName FamilyName".
```

**Comment by PRMerger9** [comment]
@dimonovdd : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/essentials/contacts.md (hunk 1 above)
Comment (dimonovdd) [reply docs/essentials/contacts.md]: I do not know, I think we can leave it at that

**Comment by davidbritch** [comment]
Is this ready to merge?

**Comment by dimonovdd** [comment]
@davidbritch the  nuget package has not been released yet. 

we should probably get a comment from @jamesmontemagno

**Comment by jamesmontemagno** [comment]
Once a nuget has been released then it can be merged in.

**Comment by jamesmontemagno** [comment]
This also needs a preview banner on it

**Comment by dimonovdd** [comment]
@jamesmontemagno 
this tag has already been added

https://docs.microsoft.com/en-us/xamarin/essentials/contacts?context=xamarin%2Fxamarin-forms&tabs=android

**Comment by dimonovdd** [comment]
@davidbritch @jamesmontemagno 

[1.6.0-pre5 nuget](https://github.com/xamarin/Essentials/releases/tag/1.6.0-pre5) is ready

**Review by davidbritch — APPROVED** [review]
Thanks @dimonovdd

**Comment by PRMerger20** [comment]
@davidbritch : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**